content/en/blog/_posts/2022-03-31-service-ip-dynamic-and-static-allocation.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: blog
+title: "Avoid collisions assigning IP to Services"
+date: 2022-03-31
```

Comment:
The release comms team will need to assign a date for this.

Reply:
where can I ping them? do they have a github alias?

Reply:
They're [#release-comms](https://kubernetes.slack.com/messages/release-comms) on Slack (and the specific person for that is Saul - I don't know a GitHub handle)